lib/teiserver/libs/teiserver_configs.ex

```diff
@@ -431,86 +431,6 @@ defmodule Teiserver.TeiserverConfigs do
       default: true
     })
 
-    add_site_config_type(%{
-      key: "teiserver.Discord channel #main",
-      section: "Discord",
-      type: "integer",
-      permissions: ["Admin"],
-      description: "The discord ID for the channel to bridge with #main"
-    })
-
-    add_site_config_type(%{
-      key: "teiserver.Discord channel #newbies",
-      section: "Discord",
-      type: "integer",
-      permissions: ["Admin"],
-      description: "The discord ID for the channel to bridge with #newbies"
-    })
-
-    add_site_config_type(%{
-      key: "teiserver.Discord channel #promote",
-      section: "Discord",
-      type: "integer",
-      permissions: ["Admin"],
-      description: "The discord ID for the channel to bridge for promoting games"
-    })
-
-    add_site_config_type(%{
-      key: "teiserver.Discord channel #overwatch-reports",
-      section: "Discord",
-      type: "integer",
-      permissions: ["Admin"],
-      description: "The discord ID for the channel to post overwatch specific reports"
-    })
-
-    add_site_config_type(%{
-      key: "teiserver.Discord channel #moderation-reports",
-      section: "Discord",
-      type: "integer",
-      permissions: ["Admin"],
-      description: "The discord ID for the channel to post moderation-reports"
-    })
-
-    add_site_config_type(%{
-      key: "teiserver.Discord channel #moderation-actions",
-      section: "Discord",
-      type: "integer",
-      permissions: ["Admin"],
-      description: "The discord ID for the channel to post moderation-actions"
-    })
-
-    add_site_config_type(%{
-      key: "teiserver.Discord channel #server-updates",
-      section: "Discord",
-      type: "integer",
-      permissions: ["Admin"],
-      description: "The discord ID for the channel to post server updates"
-    })
-
-    add_site_config_type(%{
-      key: "teiserver.Discord channel #telemetry-infologs",
-      section: "Discord",
-      type: "integer",
-      permissions: ["Admin"],
-      description: "The discord ID for the channel to post infologs"
-    })
-
-    add_site_config_type(%{
-      key: "teiserver.Discord forum #gdt-discussion",
-      section: "Discord",
-      type: "integer",
-      permissions: ["Admin"],
-      description: "The discord ID for the forum for starting GDT discussions"
-    })
-
-    add_site_config_type(%{
-      key: "teiserver.Discord forum #gdt-voting",
-      section: "Discord",
-      type: "integer",
-      permissions: ["Admin"],
-      description: "The discord ID for the forum for starting GDT voting"
-    })
```

Comment:
Nice, that got me confused for a while trying to figure out where these things lived (or not actually).